data/os/windows.yaml

```diff
@@ -5,6 +5,7 @@ mcollective::plugin_mode: ~
 mcollective::libdir: "C:/ProgramData/PuppetLabs/mcollective/plugins"
 mcollective::configdir: "C:/ProgramData/PuppetLabs/mcollective/etc"
 mcollective::rubypath: 'C:\Program Files\Puppet Labs\Puppet\sys\ruby\bin\ruby.exe'
+mcollective::facts_pidfile: "C:/ProgramData/PuppetLabs/mcollective/var/run/mcollective-facts_refresh.pid"
```

Comment:
windows does not have a `run` there, make it just `..../var/mcollective-facts_refresh.pid`